src/contour/ContourGuiApp.cpp

```diff
@@ -23,6 +25,7 @@
 #include <QtQml/QQmlContext>
 #include <QtWidgets/QApplication>
 
+#include <QtPlugin>
```

Comment:
```suggestion
#include <QtCore/QtPlugin>
```

Always have Qt includes module-prefixed. This should also fix include order.